### PR TITLE
Add example of multiworker training using TensorFlowTrainingJob

### PR DESCRIPTION
**Pull request by xyxxxxx**

### CODE REVIEW

File: job/tensorflowtrainingjob/multiworker/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# 使用 TensorFlowTrainingJob 进行多工作器同步训练
+
+本示例使用 TensorFlowTrainingJob 对 Keras 模型进行多工作器同步训练（采用 [`tf.distribute.MultiWorkerMirroredStrategy`](https://www.tensorflow.org/api_docs/python/tf/distribute/MultiWorkerMirroredStrategy) 分布式策略）。
+
```
Comment (ppkube) [inline job/tensorflowtrainingjob/multiworker/README.md:4]: pls clarify that

1. the following commands need to be executed in the Terminal of the Notebook
2. current directory is the one storing README.md

File: job/tensorflowtrainingjob/multiworker/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+执行 `download_dataset.py` 脚本以下载数据集：
+
+```shell
+python download_dataset.py
```
Comment (ppkube) [inline job/tensorflowtrainingjob/multiworker/README.md:8]:
```suggestion
# cd into the right directory
# cd ~/tutorial-examples/job/tensorflowtrainingjob/multiworker
python download_dataset.py
```

File: job/tensorflowtrainingjob/multiworker/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+```
+
+使用 `job.yaml` 创建 TensorFlowTrainingJob：
+
+```shell
+kubectl create -f job.yaml
+```
+
+前往模型构建控制台查看训练过程和结果。
```
Comment (ppkube) [inline job/tensorflowtrainingjob/multiworker/README.md:17]:
```suggestion
在命令行监控运行进展

kubectl get -o wide -w -f job.yaml

或者，前往构建控制台查看训练日志、TensorBoard 和结果等。
```

File: job/tensorflowtrainingjob/multiworker/job.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+apiVersion: batch.tensorstack.dev/v1beta1
+kind: TensorFlowTrainingJob
+metadata:
+  name: keras-mnist-trainingjob
+spec:
+  tensorboardSpec:
+    trainingLogFilesets:
+      - t9k://pvc/tutorial/job/tensorflowtrainingjob/multiworker/log
```
Comment (ppkube) [inline job/tensorflowtrainingjob/multiworker/job.yaml:8]: t9k://pvc/tutorial/tutorial-examples/job/tensorflowtrainingjob/multiworker/log

File: job/tensorflowtrainingjob/multiworker/keras_mnist_trainingjob.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        layers.Dense(64, activation='relu'),
+        layers.Dense(10, activation='softmax'),
+    ])
+    model.compile(optimizer=optimizers.Adam(learning_rate=0.001 * world_size),
+                  loss='sparse_categorical_crossentropy',
+                  metrics=['accuracy'])
+
+(train_images, train_labels), (test_images,
+                               test_labels) = datasets.mnist.load_data(
+                                   path=os.path.join(os.getcwd(), 'mnist.npz'))
```
Comment (ppkube) [inline job/tensorflowtrainingjob/multiworker/keras_mnist_trainingjob.py:49]: It's not using the pre-downloaded dataset by `job/tensorflowtrainingjob/multiworker/download_dataset.py`

File: job/tensorflowtrainingjob/multiworker/download_dataset.py [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import os
+import tensorflow as tf
+
+tf.keras.datasets.mnist.load_data(os.path.join(os.getcwd(), 'mnist.npz'))
```
Comment (ppkube) [inline job/tensorflowtrainingjob/multiworker/download_dataset.py:4]: 需要增加一个 .gitignore 以忽略： 
.ipynb_checkpoints/
log/
mnist.npz

**Review by ppkube — CHANGES_REQUESTED** [review]

File: job/tensorflowtrainingjob/multiworker/README.md (hunk 1 above)
Comment (xyxxxxx) [reply job/tensorflowtrainingjob/multiworker/README.md:4]: 1. 所有示例的命令都在 Notebook 的终端中运行，因此在根目录的 README 中声明了这一点
2. 增加了切换目录的步骤

File: job/tensorflowtrainingjob/multiworker/job.yaml (hunk 4 above)
Comment (xyxxxxx) [reply job/tensorflowtrainingjob/multiworker/job.yaml]: fixed

File: job/tensorflowtrainingjob/multiworker/keras_mnist_trainingjob.py (hunk 5 above)
Comment (xyxxxxx) [reply job/tensorflowtrainingjob/multiworker/keras_mnist_trainingjob.py]: fixed

**Review by ppkube — APPROVED** [review]

**Pull request merged**